packages/wp-story-editor/src/components/postReviewDialog/stories/postReviewDialog.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { StoryContext } from '@googleforcreators/story-editor';
+
+/**
+ * Internal dependencies
+ */
+import PostReviewDialog from '..';
+
+export default {
+  title: 'Stories Editor/Components/Dialog/Post-Review',
+  component: PostReviewDialog,
+};
+
+const storyContext = {
+  state: {
+    meta: {
+      isFreshlyPublished: true,
```

Comment:
This is currently not showing since `hasPublishAction` isn't defined.
<img width="1119" alt="Screen Shot 2022-04-27 at 4 03 16 PM" src="https://user-images.githubusercontent.com/1820266/165638709-4c2d61e9-6a5d-4ef8-be62-2cae511432d6.png">

Reply:
updating that selector where you're pulling this from should fix this.